pkg/certificate/get.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2023 The Bestchains Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package certificate
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/bestchains/bc-cli/pkg/common"
+	uhttp "github.com/bestchains/bc-cli/pkg/utils/http"
+)
+
+func NewDepositoryCertificateGetCmd(option common.Options) *cobra.Command {
+	var kid string
+
+	cmd := &cobra.Command{
+		Use:   "certificate",
+		Short: "Download the depository certificates",
+		Run: func(cmd *cobra.Command, args []string) {
+			host := viper.GetString("saas.depository.server")
+			if host == "" {
+				fmt.Fprintln(option.ErrOut, "no host provided")
+				return
+			}
+			u := fmt.Sprintf("%s%s", host, fmt.Sprintf(common.DepositoryCertificate, kid))
+			headers := map[string]string{
+				"Content-Type":        "application/octet-stream",
+				"Content-Disposition": fmt.Sprintf("attachment; filename=%s.pdf", kid),
```

Comment:
Add `Authorization` request header.

Reply:
The `NewCmd` function will always add  the `Authorization` request header.